packages/server/src/wallet/db/queries/channels.ts

```diff
@@ -64,21 +64,30 @@ async function updateAllocatorChannel(
     holdings?: Uint256;
     id?: number;
     participants?: any[];
+    guaranteedChannel: string;
   }
-  let upserts: Upsert = { channel_id: channelId, commitments, rules_address, nonce };
+  let upserts: Upsert = {
+    channel_id: channelId,
+    commitments,
+    rules_address,
+    nonce,
+    guaranteedChannel,
+  };
 
-  // For now, we just _assume_ that the channel is fully funded
-  const holdings = allocations(hubCommitment)
-    .map(x => x.amount)
-    .reduce((a, b) =>
+  // TODO: We are currently using the allocations to set the funding amount
+  // This assumes that the channel is funded and DOES NOT work for guarantor channels
```

Comment:
I'm not sure I understand this comment.

Reply:
Right now we assume the amount of funding a channel has is the sum of its `allocation`s. This assumes the channel is fully funded. 

For guarantor channels since there's no 'allocation's `holdings` will always be 0 no matter the amount of funds deposited into that channel.